src/lib/modules/api/server.ts

```diff
@@ -98,7 +98,7 @@ export default class Server {
     this.embark.events.on("plugins:register:api", (callDescription: CallDescription) => this.registerCallDescription(instance, callDescription));
 
     let ui: express.RequestHandler;
-    if (process.env.DEVELOPMENT) {
+    if (process.env.EMBARK_DEVELOPMENT) {
```

Comment:
Nope it is a env var we put ourselves.
Out of curiosity what do you have in mind?

Reply:
I know for example, for `yarn run core:install` it auto sets up an env var so that yarn doesn't prepare or something.

Was wondering if you didn't put an npm script with that too.